source/interfaceparser.py

```diff
@@ -308,6 +308,9 @@ def processfile(packname, filename, othermacros=[], timeroutines=0):
             elif text[i] == "'": i = re.search("'", text[2:]).start() + 2
             i = re.search('/', text[i:]).start() + i
             data = text[0:i+1]
+            if data.count('(')==1 and data.count(')')==1:
```

Comment:
Can you add a comment about what this is doing? I'm not sure how it works since it is doing an eval on the string between the parenthesis, but that variable is not know at this time.